spec/concerns/forms/work_form_behavior.rb

```diff
@@ -83,6 +83,17 @@
         expect(subject[:title]).to eq([stripped_title])
       end
     end
+
+    context "html in description field" do
+      let(:original_input) { '<p>Paragraph <b style="somestyle">bold</b> <a href="http://example.com">link</a><script>script</script></p>' }
```

Comment:
Can you either modify this spec or add another spec with invalid html? I know we have some records where someone has been using `<p>` as if it were `<br />` (i.e. opening but never closing) and it'd be nice to have assurance that will be resolved.